core/fabric.go

```diff
@@ -57,7 +59,7 @@ func NewFabricBase(db *db.Db) (ret *Fabric) {
 		"Enter the index the name of your default model")
 
 	ret.VendorsAll.AddVendors(openai.NewClient(), azure.NewClient(), ollama.NewClient(), groc.NewClient(),
-		gemini.NewClient(), anthropic.NewClient())
+		gemini.NewClient(), anthropic.NewClient(), dryrun.NewClient())
```

Comment:
We don't need to register the DryRun Vendor, because it is controlled directly by the separated dryRun flag. Please revert this changes.